src/Sentry.Unity/UnitySentryOptions.cs

```diff
@@ -26,6 +26,9 @@ public sealed class UnitySentryOptions
         /// </summary>
         public const string ConfigName = "SentryOptions";
 
+        public static string GetConfigPath(string? notDefaultConfigName = null)
```

Comment:
If this can be internal, it should so we can move it around without breaking consumers
```suggestion
        internal static string GetConfigPath(string? notDefaultConfigName = null)
```

Reply:
If they are internal the UnityWindow won't be able to reach it, right?

Reply:
We could make `InternalsVisibleTo` for `Sentry.Unity.Editor` so we can share API between the two without committing to public API to users of the Sentry Unity SDK